### PR TITLE
implement master mullin

### PR DESCRIPTION
**Pull request by cryogen**

### CODE REVIEW

File: server/game/cards/13.3-PoS/MaesterMullin.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const DrawCard = require('../../drawcard.js');
+const ChallengeTypes = require('../../ChallengeTypes');
+
+class MeasterMullin extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Kneel to choose a character',
+            when: {
```
Comment (ystros) [inline server/game/cards/13.3-PoS/MaesterMullin.js]: Don't need `when` key for actions.

File: server/game/cards/13.3-PoS/MaesterMullin.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                onCardEntersPlay: event => event.card === this
+            },
+            target: {
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character'
+            },
+            handler: (context) => {
+                this.selectedCharacter = context.target;
+
+                this.game.promptWithMenu(this.controller, this, {
```
Comment (ystros) [inline server/game/cards/13.3-PoS/MaesterMullin.js]: There's also an explicit "promptWithIcon" method, but this works.

File: server/game/cards/13.3-PoS/MaesterMullin.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+const DrawCard = require('../../drawcard.js');
+const ChallengeTypes = require('../../ChallengeTypes');
+
+class MeasterMullin extends DrawCard {
```
Comment (ystros) [inline server/game/cards/13.3-PoS/MaesterMullin.js]: Misspelling

File: server/game/cards/13.3-PoS/MaesterMullin.js (hunk 1 above)
Comment (cryogen) [reply server/game/cards/13.3-PoS/MaesterMullin.js]: Oops, giving away that I just copy/paste vaguely similar cards here :)

File: server/game/cards/13.3-PoS/MaesterMullin.js (hunk 2 above)
Comment (cryogen) [reply server/game/cards/13.3-PoS/MaesterMullin.js]: Are you sure? I can't find a method with that name, I just copied calm over westeros

**Pull request merged**